File: cpp/lv0/문자열_안에_문자열/문자열_안에_문자열.cpp
#include <string>
#include <vector>

using namespace std;

int solution(string str1, string str2) {
    int answer = 0;
    // std::string::find 함수는, 첫번째 인자로 문자열 내에서 찾고자 하는 문자열을 받고
    // 두번째 인자로 어디서부터 찾을지 값을 받는다. 만약 찾게 된다면 찾은 위치(pos)를 반환하고
    // 찾지 못하면 std::string::npos 객체를 반환한다.
    if (str1.find(str2, 0) != std::string::npos) {
        answer = 1;
    } else {
        answer = 2;
    }
    return answer;
}

File: go/lv0/문자열_안에_문자열/문자열_안에_문자열.go
import "strings"

func solution(str1 string, str2 string) int {
    // strings.Contains() 함수는 첫번째 인자에 두번째 인자가 포함되어 있는지 검사하여
    // 그 포함 여부를 true / false로 반환한다.
    if strings.Contains(str1, str2) == true {
        return 1
    } else {
        return 2
    }
    return 0
}